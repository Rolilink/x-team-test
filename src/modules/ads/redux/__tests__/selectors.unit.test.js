import {
  getState,
  getAds,
  getLastAd,
} from '../selectors';

const state = {
  ads: {
    ads: [1, 2, 3],
  },
};

describe('Ads.Redux.Selectors.getState', () => {
  it('should return the ads state', () => {
    expect(getState(state)).toMatchSnapshot();
  });
});

describe('Ads.Redux.Selectors.getAds', () => {
  it('should return the ads array', () => {
    expect(getAds(state)).toMatchSnapshot();
  });
});

describe('Ads.Redux.Selectors.getLastAd', () => {
  it('should return the last ad in the ads array', () => {
    expect(getLastAd(state)).toMatchSnapshot();
  });
});
