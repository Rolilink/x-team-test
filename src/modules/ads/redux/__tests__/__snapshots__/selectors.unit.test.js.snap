// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ads.Redux.Selectors.getAds should return the ads array 1`] = `
Array [
  1,
  2,
  3,
]
`;

exports[`Ads.Redux.Selectors.getLastAd should return the last ad in the ads array 1`] = `3`;

exports[`Ads.Redux.Selectors.getState should return the ads state 1`] = `
Object {
  "ads": Array [
    1,
    2,
    3,
  ],
}
`;
